tldw_Server_API/app/core/Setup/audio_bundle_catalog.py

```
@@ -0,0 +1,325 @@
+    model_assets: list[AudioBundleStep] = Field(default_factory=list)
+    resource_profiles: dict[str, AudioResourceProfile] = Field(default_factory=dict)
+    stt_plan: list[dict[str, Any]] = Field(default_factory=list)
+    tts_plan: list[dict[str, Any]] = Field(default_factory=list)
+    embeddings_plan: dict[str, list[str]] = Field(
+        default_factory=lambda: {"huggingface": [], "custom": [], "onnx": []}
+    )
+    default_config_updates: dict[str, dict[str, str]] = Field(default_factory=dict)
+    verification_targets: list[str] = Field(default_factory=list)
+
+    @model_validator(mode="after")
+    def populate_profile_compatibility_fields(self) -> "AudioBundle":
```

Ruff (0.15.6)

[warning] Remove quotes from type annotation

Remove quotes

(UP037)